tests/verkle/eip4762_verkle_gas_witness/test_contract_execution.py

```diff
@@ -79,126 +82,126 @@ def code_with_jumps(size, jumps: list[Jump | Jumpi] = []):
+        #     42,
+        #     [[0, 0], [33, 33]],
+        # ),
+        # (  # jump_outside_code_size
+        #     code_with_jumps(150 * 31, [Jump(50, 150 * 31 + 42)]),
+        #     1_000_000,
+        #     [[0, 0]],
+        # ),
+        # (  # jumpi_outside_code_size
+        #     code_with_jumps(150 * 31, [Jumpi(50, 150 * 31 + 42, True)]),
+        #     1_000_000,
+        #     [[0, 0]],
+        # ),
+        # (  # push20 with data split in two chunks
+        #     Op.PUSH0 * (31 - (1 + 10)) + Op.PUSH20(0xAA),
+        #     1_000_000,
+        #     [[0, 1]],
+        # ),
+        # (  # push32 spanning three chunks
+        #     Op.PUSH0 * (31 - 1) + Op.PUSH32(0xAA),
+        #     1_000_000,
+        #     [[0, 2]],
+        # ),
+        # (  # pushn with expected data past code size
+        #     Op.PUSH0 * (31 - 5) + Op.PUSH20,
+        #     1_000_000,
+        #     [[0, 0]],
+        # ),
     ],
     ids=[
         "only_code_in_account_header",
-        "chunks_both_in_and_out_account_header",
-        "touches_only_first_byte_code_chunk",
-        "touches_only_last_byte_code_chunk",
-        "pushn_with_data_in_chunk_that_cant_be_paid",
-        "jump_to_jumpdest_in_pushn_data",
-        "jumpi_to_jumpdest_in_pushn_data",
-        "jump_to_non_jumpdest_destiny",
-        "jumpi_to_non_jumpdest_destiny",
-        "linear_execution_stopping_at_first_byte_of_next_chunk",
-        "false_jumpi",
-        "insufficient_gas_for_jump_instruction",
-        "insufficient_gas_for_jumpi_instruction",
-        "sufficient_gas_for_jump_instruction_but_not_for_code_chunk",
-        "sufficient_gas_for_jumpi_instruction_but_not_for_code_chunk",
-        "jump_outside_code_size",
-        "jumpi_outside_code_size",
-        "push20_with_data_split_in_two_chunks",
-        "push32_spanning_three_chunks",
-        "pushn_with_expected_data_past_code_size",
+        # "chunks_both_in_and_out_account_header",
+        # "touches_only_first_byte_code_chunk",
+        # "touches_only_last_byte_code_chunk",
+        # "pushn_with_data_in_chunk_that_cant_be_paid",
+        # "jump_to_jumpdest_in_pushn_data",
+        # "jumpi_to_jumpdest_in_pushn_data",
+        # "jump_to_non_jumpdest_destiny",
+        # "jumpi_to_non_jumpdest_destiny",
+        # "linear_execution_stopping_at_first_byte_of_next_chunk",
+        # "false_jumpi",
+        # "insufficient_gas_for_jump_instruction",
+        # "insufficient_gas_for_jumpi_instruction",
+        # "sufficient_gas_for_jump_instruction_but_not_for_code_chunk",
+        # "sufficient_gas_for_jumpi_instruction_but_not_for_code_chunk",
+        # "jump_outside_code_size",
+        # "jumpi_outside_code_size",
+        # "push20_with_data_split_in_two_chunks",
+        # "push32_spanning_three_chunks",
+        # "pushn_with_expected_data_past_code_size",
     ],
 )
-def test_contract_execution(
+def test_contract_execution_foo(
```

Comment:
Quick temporary name to run more easily discarding other stuff:
`fill --fork Verkle -v -m blockchain_test -n auto --evm-bin=/data/kaustinen/go-ethereum/evm -k test_contract_execution_foo`